Readme.md

```diff
@@ -7,7 +7,7 @@
 [![devDependency Status](https://david-dm.org/socketio/socket.io/dev-status.svg)](https://david-dm.org/socketio/socket.io#info=devDependencies)
 [![NPM version](https://badge.fury.io/js/socket.io.svg)](https://www.npmjs.com/package/socket.io)
 ![Downloads](https://img.shields.io/npm/dm/socket.io.svg?style=flat)
-[![](http://slack.socket.io/badge.svg?)](http://slack.socket.io)
+[![](https://img.shields.io/badge/Slack-socket.io-brightgreen.svg?style=flat)](https://slackin-socketio.now.sh)
```

Comment:
Yeah i noticed that after making the PR. Will update. You must understand that took a bit of investigating to even figure out what was going on. Since it was the subdomain that was erroring. I figured better to have a finished stick figure than an unfinished Picasso :-). Updating now @darrachequesne.